src/backend/src/controllers/users.controllers.ts

```diff
@@ -104,3 +105,45 @@ export const logUserIn = async (req: any, res: any) => {
 
   return res.status(200).json(authenticatedUserTransformer(user));
 };
+export const updateUserRole = async (req: any, res: any) => {
+  const updatingUserId: number = parseInt(req.params.userId);
+
```

Comment:
also need to make the check here that express validator didn't find any errors with the body